pkg/cmd/tracee.go

```diff
@@ -88,15 +88,30 @@ func (r Runner) Run(ctx context.Context) error {
 		}
 	}()
 
+	// Run Tracee
+
+	if r.Printer.Active() {
```

Comment:
In the future, when we can dynamically add or remove streams, won't this prevent adding streams after starting with "ignore"?

Reply:
Possibly, but to reach that point a redesign will be required anyway.